<?xml version="1.0"?>
<robot name="zedm">
  <link name="zedm_base_link">
    <visual>
      <origin xyz="0 0 0" rpy="0 0 0"/>
      <geometry>
        <box size="0.1 0.05 0.05"/>
      </geometry>
      <material name="black">
        <color rgba="0.0 0.0 0.0 1.0"/>
      </material>
    </visual>
  </link>
  <joint name="zedm_joint" type="fixed">
    <parent link="tool0"/>
    <child link="zedm_base_link"/>
    <origin xyz="0 0 0.1" rpy="0 0 0"/>
  </joint>
</robot>

